saleor/plugins/webhook/tests/test_payment_webhook_utils.py

```diff
@@ -178,7 +178,7 @@ def test_parse_payment_action_response_parse_amount(
     assert gateway_response.amount == dummy_webhook_app_payment_data.amount
 
 
-def test_clear_successful_delivery(event_delivery):
+def test_clear_successful_delivery(event_delivery, event_payload):
```

Comment:
you should take payload from `event_delivery.payload` not from another fixture

Reply:
done